### PR TITLE
Reduce noisy tendermint logs

### PR DESCRIPTION
**Pull request by yzang2019**
## Describe your changes and provide context
This change will help to reduce the noisy logs such as
image

### CODE REVIEW

File: internal/consensus/state.go [hunk 1]
```diff
@@ -2003,7 +2003,7 @@ func (cs *State) enterCommit(ctx context.Context, height int64, commitRound int3
 	logger := cs.logger.With("height", height, "commit_round", commitRound)
 
 	if cs.roundState.Height() != height || cstypes.RoundStepCommit <= cs.roundState.Step() {
-		logger.Info(
+		logger.Debug(
```
[the remark this answers is not in the document]
Comment (yzang2019) [reply internal/consensus/state.go:2006]: good point

[the remark this answers is not in the document]
Comment (yzang2019) [reply internal/consensus/state.go:2006]: Fixed, check again?

**Review by codchen — APPROVED** [review]
just a q: after this change will there be any log left that can indicate consensus is progressing?

**Comment by yzang2019** [comment]
> just a q: after this change will there be any log left that can indicate consensus is progressing?

Yeah, there'll still be logs around commit info such as:
image

**Review by philipsu522 — APPROVED** [review]

**Pull request merged**